engine/src/main/resources/templates/Common/insightsEmailBody.html

```diff
@@ -0,0 +1,521 @@
+                                                        <span class="rh-masthead__section__head__subtitle">{#insert content-subtitle-section2}{/}</span>
+                                                    </p>
+                                                </td>
+                                                <td class="rh-masthead__section__head">
+                                                    <p><span class="rh-masthead__section__head__counter">{#insert content-title-right-part-section2}{/}</span>
+                                                </td>
+                                            </tr>
+                                        </table>
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td class="rh-content__block" style="padding: 0px 0px 24px 0px;"/>
+                                </tr>
+                                <tr>
+                                    <td class="rh-content__block">
+                                        {#insert content-body-section2}{/}
+                                    </td>
+                                </tr>
+                                {#if renderButtonSection2}
+                                <tr>
+                                    <td class="rh-content__block" style="padding: 24px 0px;">
+                                        <table align="center">
+                                            <tr>
+                                                <td class="rh-cta-link" align="center">
+                                                    {#insert content-button-section2}{/}
+                                                </td>
+                                            </tr>
+                                        </table>
+                                    </td>
+                                </tr>
+                                {#else}
+                                <tr>
+                                    <td class="rh-content__block" style="padding: 0px 0px 24px 0px;"/>
+                                </tr>
+                                {/if}
+                            </table>
+                        </td>
+                    </tr>
+                    {/if}
+
+                    <!-- end body section -->
+
+                    <!-- Footer -->
+                    <tr>
+                        <td class="rh-footer">
+                            <table role="presentation" border="0" align="center" cellpadding="0" cellspacing="0" width="100%">
+                                <tr>
+                                    <td class="rh-footer__text">
+                                        This email was sent by Hybrid Cloud Console | <a href="{environment.url}/user-preferences/notifications/{action.bundle}" target="_blank">Manage email preferences</a>
```

Comment:
There are at least 4 different versions of that footer in the existing templates:

> This email was sent by Red Hat Insights |<a href="{environment.url}/user-preferences/notifications/openshift" target="_blank">&nbsp;Manage email preferences</a>

> This email was sent by Red Hat Insights |<a href="{environment.url}/user-preferences/notifications/rhel" target="_blank">&nbsp;Manage notification preferences</a>

> You received this notification because of selections you made in your <a href="{environment.url}/user-preferences/notifications/rhel" target="_blank">Hybrid Cloud Console User Preferences</a>. You can opt in or out of receiving notifications by changing your User Preferences.

> This email was sent by Red Hat OpenShift Streams for Apache Kafka |<a href="{environment.url}/user-preferences/notifications/application-services" target="_blank">&nbsp;Manage notification preferences</a>

Does the one from this PR come from the UXD guidelines?

Reply:
according UXD guidelines:
_Footer will always be the same
This email was sent by Hybrid Cloud Console | Manage email preferences
Text Color -
White - PF Color 1 --pf-global--BackgroundColor--100
Blue - PF Color 17 --pf-global--info-color--100
Banner - PF Color 11 --pf-global--BackgroundColor--dark-300_